app/src/main/java/net/gsantner/markor/format/asciidoc/AsciidocSyntaxHighlighter.java

```diff
@@ -0,0 +1,324 @@
+        } else {
+//          createColorSpanForMatches(HEADING_SIMPLE, _isDarkMode ? AD_COLOR_DARK_HEADING : AD_COLOR_LIGHT_HEADING);
+            createSpanForMatches(HEADING_SIMPLE, new HighlightSpan().setForeColor(_isDarkMode ? AD_FORECOLOR_DARK_HEADING : AD_FORECOLOR_LIGHT_HEADING));
+        }
+
+        if (_highlightCodeChangeFont) {
+            // TODO: How to use setTypeface in createSpanForMatches to set MONOSPACE?
+            // setTypeface(Typeface.MONOSPACE) - invalid
+            // setTypeface(0) - doesn't work
+            // but the current solution works fine and allows to use MONOSPACE based on settings
+            createMonospaceSpanForMatches(MONOSPACE);
+            createMonospaceSpanForMatches(BLOCK_DELIMITED_LISTING);
+            createMonospaceSpanForMatches(BLOCK_DELIMITED_LITERAL);
+            createMonospaceSpanForMatches(LIST_UNORDERED);
+            createMonospaceSpanForMatches(LIST_ORDERED);
+            createMonospaceSpanForMatches(LIST_DESCRIPTION);
+            createMonospaceSpanForMatches(ATTRIBUTE_DEFINITION);
+            createMonospaceSpanForMatches(ATTRIBUTE_REFERENCE);
+            createMonospaceSpanForMatches(ADMONITION);
+            createMonospaceSpanForMatches(SQUAREBRACKETS);
+        }
+
+//      createStyleSpanForMatches(BOLD, Typeface.BOLD);
+        createSpanForMatches(BOLD, new HighlightSpan().setBold(true));
+        createSpanForMatches(ITALICS, new HighlightSpan().setItalic(true));
+
+        createSpanForMatches(LINK_PATTERN, new HighlightSpan().setForeColor(_isDarkMode ? AD_FORECOLOR_DARK_LINK : AD_FORECOLOR_LIGHT_LINK));
+        createSpanForMatches(XREF_PATTERN, new HighlightSpan().setForeColor(_isDarkMode ? AD_FORECOLOR_DARK_LINK : AD_FORECOLOR_LIGHT_LINK));
+        createSpanForMatches(IMAGE_PATTERN, new HighlightSpan().setForeColor(_isDarkMode ? AD_FORECOLOR_DARK_LINK : AD_FORECOLOR_LIGHT_LINK));
+        createSpanForMatches(INCLUDE_PATTERN, new HighlightSpan().setForeColor(_isDarkMode ? AD_FORECOLOR_DARK_LINK : AD_FORECOLOR_LIGHT_LINK));
+        createSpanForMatches(LIST_UNORDERED, new HighlightSpan().setForeColor(_isDarkMode ? AD_FORECOLOR_DARK_LINK : AD_FORECOLOR_LIGHT_LINK));
+        createSpanForMatches(LIST_ORDERED, new HighlightSpan().setForeColor(_isDarkMode ? AD_FORECOLOR_DARK_LINK : AD_FORECOLOR_LIGHT_LINK));
+        createSpanForMatches(LIST_DESCRIPTION, new HighlightSpan().setForeColor(_isDarkMode ? AD_FORECOLOR_DARK_LINK : AD_FORECOLOR_LIGHT_LINK));
+
+        //TODO: test for interference with other role like underline, line-through => interference
+        // to aggressive
+        // createColorSpanForMatches(ROLE_GENERAL, AD_COLOR_ROLE_GENERAL);
+
+        // DONE: use createSpanForMatches() to change multiple settings in one call
+        // createStyleSpanForMatches(ADMONITION, Typeface.BOLD);
+        // createColorSpanForMatches(ADMONITION, AD_COLOR_ADMONITION);
```

Comment:
noted as done but has some commented code? remove it

Reply:
removed